src/services/Wishlist/removeItemFromWishlist.js

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+const removeItemFromWishlist = async (product, authDispatchFunction, authStateGiven) => {
+    try {
+        const {
+            data: {
+                wishlist: wishlistUpdated
+            }
+        } = await axios.delete(`/api/user/wishlist/${product._id}`, {
+            headers: {
+                authorization: authStateGiven.authenticationToken
+            }
+        })
+        authDispatchFunction({ type: "WISHLIST", payload: wishlistUpdated })
+    } catch (error) {
+        console.log(error);
```

Comment:
you can remove this log

Reply:
Used for error handling so cannot remove it!!